File: development/prime_number/PrimeChecker.cs
using System;

/*class Program
{

    static bool IsPrime(int num)
    {
        if(num <= 1) return false;
        for (int i = 2; i <= Math.Sqrt(num); i++)
        {
            if(num % i == 0) return false;
        }
        return true;
    }

    static void Main()
    {
        int num1 = 29;
        int num2 = 30;
        Console.WriteLine("The number: " + num1 + " is prime: " + IsPrime(num1));
        Console.WriteLine("The number: " + num2 + " is prime: " + IsPrime(num2));
    }
}*/


bool IsPrime(int num)
{
    if(num <= 1) return false;
    for (int i = 2; i <= Math.Sqrt(num); i++)
    {
        if(num % i == 0) return false;
    }
    return true;
}

int num1 = 29;
int num2 = 30;
Console.WriteLine("The number: " + num1 + " is prime: " + IsPrime(num1));
Console.WriteLine("The number: " + num2 + " is prime: " + IsPrime(num2));

File: development/prime_number/prime_checker.py
import math

def is_prime(num):
    if num <= 1:
        return False
    for i in range(2, int(math.sqrt(num)) + 1):
        if num % i == 0:
            return False
    return True

num1 = 43
num2 = 45

print("the number" + num1 + " is prime: " + is_prime(num1))
print("the number" + num2 + " is prime: " + is_prime(num2))